doc/configuration.rst

```diff
@@ -493,36 +495,60 @@ dictionary following the pattern below::
          # Required keys
          'org': '<github_org>',
          'repo': '<github_repo>',
-         'url': '<binder_url>',  # Any URL of a binder server. Must be full URL (e.g. https://mybinder.org).
-         'branch': '<branch-for-documentation>',  # Can be any branch, tag, or commit hash. Use a branch that hosts your docs.
+         'ref': '<ref-for-documentation>',  # Can be any branch, tag, or commit hash. Use a branch that hosts your docs.
+         'binderhub_url': '<binder_url>',  # Any URL of a binderhub deployment. Must be full URL (e.g. https://mybinder.org).
          'dependencies': '<list_of_paths_to_dependency_files>',
          # Optional keys
-         'filepath_prefix': '<prefix>' # A prefix to append to any filepaths in Binder links.
-         'notebooks_dir': '<notebooks-directory-name>' # Jupyter notebooks for Binder will be copied to this directory (relative to site root).
+         'filepath_prefix': '<prefix>' # A prefix to prepend to any filepaths in Binder links.
+         'notebooks_dir': '<notebooks-directory-name>' # Jupyter notebooks for Binder will be copied to this directory (relative to built documentation root).
          'use_jupyter_lab': <bool> # Whether Binder links should start Jupyter Lab instead of the Jupyter Notebook interface.
          }
     }
 
-Note that ``branch:`` should be the branch on which your documentation is hosted.
-If you host your documentation on GitHub, this is usually ``gh-pages`` or ``master``.
+If a Sphinx-Gallery configuration for Binder is discovered, the following extra things will happen:
+
+1. The dependency files specified in ``dependencies`` will be copied to a ``binder/`` folder in your built documentation.
+2. The built Jupyter Notebooks from the documentation will be copied to a folder called ``<notebooks_dir/>`` at the root of
+   your built documentation (they will follow the same folder hierarchy within the notebooks directory folder.
+2. The rST output of each Sphinx-Gallery example will now have a ``launch binder`` button in it.
+4. That button will point to a binder link with the following structure::
+
+       <binderhub_url>/v2/gh/<org>/<repo>/<ref>?filepath=<filepath_prefix>/<notebooks_dir>/path/to/notebook.ipynb
+
+Below is a more complete explanation of each field.
+
+**org** (type: string)
```

Comment:
no need for `**` to bold here, def lists become bold already

https://667-25860190-gh.circle-artifacts.com/0/html/configuration.html